### ISSUE TITLE
Proposal to release 2.0-alpha7 with bug fixes and awaited features for Kubeflow 1.7 release

### ISSUE THREAD

**Issue by surajkota** [kubeflow/pipelines#8938]
Hi @zijianjoy @chensun, I am opening this issue to request for releasing a patch on top of 2.0-alpha.6 instead of using 2.0-alpha.6 release for Kubeflow-1.7 release. [2.0-alpha6 was released 5 months ago](https://github.com/kubeflow/pipelines/releases/tag/2.0.0-alpha.6) which is in the same timeframe as [Kubeflow-1.6 release](https://github.com/kubeflow/manifests/releases/tag/v1.6.1) and considering the frequency of Kubeflow releases, users have to wait a long time for some of the big fixes and features which have been merged since then. Following are the issues/PRs I propose to be made available in the pipelines version included in 1.7 Kubeflow release (maybe 2.0-alpha7):

1. (Regression in Kubeflow-1.6.1) https://github.com/kubeflow/pipelines/pull/8343
1. (Regression in Kubeflow-1.7.rc1) https://github.com/kubeflow/pipelines/pull/8553. This has been reported in RC1 testing, see https://github.com/kubeflow/manifests/issues/2402. Although 10 days/this fix does not seem to be sufficient
    - > suggesting to completely disabling binlogs using '--disable-log-bin'. [Setting 'binlog-expire-logs' to 10 days, as set in pipelines](https://github.com/kubeflow/pipelines/blob/master/manifests/kustomize/third-party/mysql/base/mysql-deployment.yaml) didn't work for me as the storage ran full much sooner than that
1. (Feature) https://github.com/kubeflow/pipelines/pull/8651 solves a long standing [feature request](https://github.com/kubeflow/pipelines/issues/3405) and we/AWS would like to make it available in 1.7
1. (Fix/feature) https://github.com/kubeflow/pipelines/pull/8401
1. PRs which are updating deprecated and vulnerable packages e.g. ones opened by dependabot with `chore(deps)`

(reduce scope because of WG bandwidth)~~Nice to have:~~
~~-(Feature) https://github.com/kubeflow/pipelines/pull/8511. @kimwnasptd to comment if this is much awaited feature~~
~~- https://github.com/kubeflow/pipelines/pull/8384~~
~~- https://github.com/kubeflow/pipelines/pull/8793~~

I have compiled this list based on customer contacts we received and looking at feat/fix(frontend) or feat/fix(backend) related to KFP v1 from https://github.com/kubeflow/pipelines/compare/2.0.0-alpha.6...2.0.0-beta.0. 

Note: I have skipped sdk related PRs assuming user can update sdk independently and it does not require backend changes but I might have missed some w.r.t. this because I did not go through each PR description

@kubeflow/release-team @kubeflow/release-managers @DomFleischmann

@kimwnasptd since he wanted to add some PRs they are waiting on. Please add if I missed them

Please let me know your thoughts

**Comment by surajkota** [comment]
Pipelines WG is working on a patch release on top of alpha6 which will also include another fix for sql logs issue(https://github.com/kubeflow/pipelines/pull/8941). They have limited bandwidth right now and considering the recent code refractor, release cycle timeframe, they are able to include only the first 3 items for the patch release.

Appreciate the bias for action from Chen and pipelines WG for helping with resolution of these issues in 1.7 release

cc @DomFleischmann can we plan for RC2 when this release is out?

**Comment by chensun** [comment]
Created the release branch and cherry-picked #8343 and #8651, currently waiting for #8941 to be merged into master first.
I didn't encounter any merge conflict during the cherrypick so far.

https://github.com/kubeflow/pipelines/commits/release-2.0-alpha

**Comment by DomFleischmann** [comment]
@chensun What is the state of this new release? We are on a very tight schedule with the 1.7 release supposedly happening on March 15th. I'm willing to delay this date but we need something as soon as possible to give people enough time to test.

**Comment by surajkota** [comment]
Tag was cut yesterday, created a PR to sync the manifests - https://github.com/kubeflow/manifests/pull/2404/files

Thanks @chensun. Assuming we are good to use the tag for next RC, let us know this is not the case

**Comment by chensun** [comment]
Yes, tag was cut yesterday, and release is now published: https://github.com/kubeflow/pipelines/releases/tag/2.0.0-alpha.7
